source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1198,6 +1199,18 @@ message TransformationSwapCommutableOperands {
 
 }
 
+message TransformationSwapConditionalBranchOperands {
+
+  // Swaps label ids in OpBranchConditional instruction.
```

Comment:
add that it inverts the guard too (and maybe also mention branch weights).

Reply:
Fixed in b08e5a3.